spec/src/main/doc/events.asciidoc

```diff
@@ -451,6 +523,10 @@ The order of more than one observer with the same priority is undefined and the
 
 The transaction context, client security context and lifecycle contexts active when an observer method is invoked depend upon what kind of observer method it is.
 
+* If the observer method is asynchronous, it is called:
+ ** in the same client security context as invocation of `Event.fireAsync()` or `BeanManager.fireAsync()`, and
+ ** in an unspecified transaction context, and
+ ** within a new lifecycle for request context, session context and conversation context. 
```

Comment:
Yes, we can say they are dummy. Do you suggest to forbid (exception thrown) injection of bean in these scopes?